Sources/MapboxNavigation/VisualInstruction.swift

```diff
@@ -121,5 +121,52 @@ extension VisualInstruction {
         
         return instructionLabel.attributedText
     }
+
+    /// Returns a `CPImageSet` representing the maneuver lane configuration.
+    @available(iOS 12.0, *)
+    public func lanesImageSet(side: DrivingSide, direction: ManeuverDirection?, scale: CGFloat)  -> CPImageSet? {
+        // create lanes visual banner
+        // The `lanesImageMaxSize` size is an estimate of the CarPlay Lane Configuration View
+        // The dimensions are specified in the CarPlay App Programming Guide - https://developer.apple.com/carplay/documentation/CarPlay-App-Programming-Guide.pdf
```

Comment:
Link to the specific page for convenience:

https://developer.apple.com/carplay/documentation/CarPlay-App-Programming-Guide.pdf#page=38

Reply:
Done.